source/docs/software/wpilib-tools/robot-characterization/generating-project.rst

```diff
@@ -49,6 +37,8 @@ Accordingly, you can also load an existing config file (config files are project
 .. image:: images/loading-config-file.png
    :alt: Loading a saved configuration file
 
+.. note:: Now is also a good time to set the Team Number box and modify the units and units per rotation if necessary (units per rotation is NOT equivalent to the wheel diameter from last year's tool). These units should facilitate a rough validation of the measurements recorded by the robot (e.g. using rotations for a flywheel to see if the recorded rotations seem to match what you saw).
```

Comment:
We should explain what "units" and "units per rotation" mean in this context. For example, is it a conversion from rotations of the motor shaft to meters along the ground? Is it something else? A concrete example or two may clarify this.

Reply:
Is this a better explanation?

Now is also a good time to set the Team Number box and modify the units and units per rotation if necessary (units per rotation is NOT equivalent to the wheel diameter from last year's tool). The units field lets you choose between various rotational and translational units (rotations, radians, degrees, feet, meters, and inches). You should choose a unit that facilitates a rough validation of the recorded measurements. For example, you could choose rotations when testing a flywheel to see if the tool's recorded rotations seem reasonable, or you could choose feet when testing a drive base to see if the recorded distance seems reasonable. On the other hand, the units per rotation entry is how many of the previously specified units are recorded per rotation of the shaft. Do note that rotational units (rotations, radians, and degrees) have predefined/unmutable units per rotations. In contrast, translational units (meters, feet, inches) require that you specify the conversion, such as a wheel with a 3-inch diameter can be converted to 9.42 inches per rotation (pi * 3).

Reply:
That's quite lengthy for a ``.. note::``, just turn it into a regular paragraph at that point.

Reply:
I added it under its own [section](https://github.com/wpilibsuite/frc-docs/blob/620dac216667bd4195ac993a11b5a71110aa9659/source/docs/software/wpilib-tools/robot-characterization/generating-project.rst). Is it more clear now?